MsGraphicsPkg/Library/FrameBufferMemDrawLib/FrameBufferMemDrawLibPei.inf

```diff
@@ -0,0 +1,43 @@
+
+## @file
+# A library for drawing directly to the FrameBuffer memory region
+#
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+
+[Defines]
+  INF_VERSION    = 0x00010017
+  BASE_NAME      = FrameBufferMemDrawLib
+  FILE_GUID      = 23E999E3-B231-4076-8791-DA5151BB3DC4
+  VERSION_STRING = 1.0
+  MODULE_TYPE    = PEIM
+  LIBRARY_CLASS  = FrameBufferMemDrawLib

```

Comment:
Would you need the destructor as well?

Reply:
Since this instance will be used only until end of pei,  I dont think it is required.